sys-libs/ncurses-compat-5.9 will not build with lto

CFLAGS="-march=native -O2 ${FLTO} -fuse-linker-plugin -pipe -ftree-vectorize"

checking whether able to link to dl*() functions... configure: error: Cannot link test program for libdl

--- a/sys-config/ltoize/files/package.cflags/ltoworkarounds.conf
+++ b/sys-config/ltoize/files/package.cflags/ltoworkarounds.conf
@@ -114,6 +114,7 @@ media-libs/ilmbase *FLAGS+=-lrt # needed for linker to build media-libs/openexr 
 x11-libs/wxGTK NOLDADD=1 C*FLAGS-="-Wl,*" # Issue #40, does not build when -Wl linker flags are passed to the compiler through C*FLAGS
 #dev-qt/qtcore "export EXTRA_CXXFLAGS='${CXXFLAGS}' EXTRA_LFLAGS='${LDFLAGS}'" #work around overwritten flags (#32)
 sys-libs/ncurses *FLAGS-=-Wl,--as-needed *FLAGS+=-ldl # checking whether able to link to dl*() functions... configure: error: Cannot link test program for libdl (#111)
+sys-libs/ncurses-compat *FLAGS-=-Wl,--as-needed *FLAGS+=-ldl # checking whether able to link to dl*() functions... configure: error: Cannot link test program for libdl (#111)
 kde-apps/dolphin *FLAGS-="-Wl,--as-needed" # ( https://github.com/InBetweenNames/gentooLTO/issues/50 )
 
 sys-power/thermald *FLAGS+=-Wl,--no-fatal-warnings *FLAGS+=-Wno-error

